### ISSUE TITLE
Using Python to connect to an aql database with pyodbc

### ISSUE THREAD

**Issue by chivian** [chivian/Data-Science-Machine-Learning-Articles#11]
## TL;DR
Connecting to an sql database with python using pyodbc
### Article Link
https://datatofish.com/how-to-connect-python-to-sql-server-using-pyodbc/
### Author
Data fish
## Key Takeaways
* The pyodbc package enables a seamless connection to an sql database using pyuthon
* 

## Useful Code Snippets
```python
#Add your codes here
import numpy as np
import pandas as pd

```
import pyodbc 

conn = pyodbc.connect('Driver={SQL Server};'
                      'Server=server_name;'
                      'Database=database_name;'
                      'Trusted_Connection=yes;')

cursor = conn.cursor()
cursor.execute('SELECT * FROM table_name')

for i in cursor:
    print(i)

## Useful Tools
* pyodbc
* Jupyter notebook
* sql database
